PowerValidatedSolutions.psm1

```diff
@@ -48145,6 +48145,383 @@ Function Test-WMSubnetInput {
 }
 Export-ModuleMember -Function Test-WMSubnetInput
 
+Function Invoke-GeneratePrivateKeyAndCsrFiles {
+    <#
+        .SYNOPSIS
+        Generate Private Key and CSR files using Openssl, common name and neccesary parameters.  
+
+        .DESCRIPTION
+        The Invoke-GeneratePrivateKeyAndCsrFiles cmdlet generate private key and CSR files using Openssl, common name and input parameters.  
+        
+        .EXAMPLE
+        Invoke-GeneratePrivateKeyAndCsrFiles -certDir "C:\certificates" -commonName "xint-vrli01.rainpole.io" -subjectAlternativeNames 
+        "xint-vrli01a.rainpole.io, xint-vrli01b.rainpole.io, xint-vrli01c.rainpole.io" -keySize 4096 -expireDays 720 -organization rainpole 
+        -organizationUnit "IT" -locality "San Francisco" -state "California" -country "US"
+        This example will generate a private key and csr pair for a VIP xint-vrli01.rainpole.io with cluster nodes xint-vrli01a, xint-vrli01b 
+        and xint-vrli01c as SAN values.  
+
+        .PARAMETER certDir
+        The directory path to store the SSL certificate private key and certificate signing request file.
```

Comment:
```suggestion
        The directory path to store the signed certificate, private key and certificate signing request file.
```

Reply:
Addressed in commit 53039acbaaffafaf219274cac1a3684735130f53